hack/golangci-lint.sh

```diff
@@ -3,6 +3,23 @@
 # Need to run linter twice to cover all the build tags code paths
 set -e
 
+# Mac's are "special" and don't support `declare -A`
+if [[ $(uname -s) == "Darwin" ]] || [[ "$GOOS" == "darwin" ]]; then
+  declare -a DARWIN_SKIP_DIRS
+  DARWIN_SKIP_DIRS=(
+    pkg/api
+    pkg/domain/infra/abi
+    test
+    pkg/machine/qemu
+    libpod/events
+    pkg/trust
+  )
+  echo "Skipped directories: ${DARWIN_SKIP_DIRS[*]}"
```

Comment:
There's no `-x` in this script. To help future spelunkers, would you consider making this the same as lines 37-39 below? Something to show up in the log as "running lint"?

Reply:
ya, sure, np.